clusterloader2/pkg/measurement/interface.go

```diff
@@ -35,6 +35,7 @@ type MeasurementConfig struct {
 	// TemplateProvider provides templated objects.
 	TemplateProvider    *config.TemplateProvider
 	ClusterLoaderConfig *config.ClusterLoaderConfig
+	MeasurementManager  MeasurementManager
```

Comment:
Could you extract all the changes related to introducing MeasurementManager into a separate PR?
It's a good refactoring that would be worth to do anyway

Reply:
hey @mm4tt i have created a separate PR for these changes:
https://github.com/kubernetes/perf-tests/pull/1079